smos-docs-site/pages/concepts.markdown

```diff
@@ -0,0 +1,70 @@
+---
+title: SMOS Concepts
+date: 2020-04-08
+---
+
+GTD
+: GTD is a time management method by David Allen (see [Getting This
+  Done](https://en.wikipedia.org/wiki/Getting_Things_Done) on
+  Wikipedia).
+  SMOS is supposed to be a tool to implement GTD.
+
+
+<!-- the next concepts are related to SMOS configuration, but AFAIK
+are not explained -->
+
+Workflow
+: What's the meaning in SMOS and why the need for a directory?
+
+
+Projects
+: Whats the meaning in SMOS?  Why the need for a directory in the
+  configuration file?
+
+
+Archive:
+: When task and projects are completed, they are archived (i.e. moved
+  in the archive directory).
+
+State File
+: ???
```

Comment:
I'm not sure where you got this?
There are some state files for the sync client, but let's be clear first.

Reply:
In the `smos-scheduler` help screen.  I was curious about the scheduler :-)

Reply:
Aha! The scheduler doesn't actually work yet so let's ignore it for now.